actors/builtin/power/power_state.go

```diff
@@ -61,55 +60,6 @@ func ConstructState(emptyMapCid, emptyMMapCid cid.Cid) *State {
 	}
 }
 
-// Note: this method is currently (Feb 2020) unreferenced in the actor code, but expected to be used to validate
-// Election PoSt winners outside the chain state. We may remove it.
-func (st *State) minerNominalPowerMeetsConsensusMinimum(s adt.Store, miner addr.Address) (bool, error) {
```

Comment:
I'm biased towards removing everything that is unused, but not at all confident deleting is the best call since I don't know any of the stories behind unused methods.  This one looks like it belongs in lotus rather than here.

Reply:
I've probably deleted other things I shouldn't have so look closely at red methods

Reply:
This one seems like a good one to get rid of.

Reply:
Please keep this one and point to https://github.com/filecoin-project/specs-actors/issues/266, for which it is documenting our intentions.